lib/logic/player/music_player.dart

```diff
@@ -531,45 +529,44 @@ class AudioHandler extends BaseAudioHandler with SeekHandler, WidgetsBindingObse
     final l10n = staticl10n;
     playbackState.add(playbackState.value!.copyWith(
       controls: [
-        // TODO: currently using custom API from my fork, see https://github.com/ryanheise/audio_service/issues/633
```

Comment:
The issue remains open. Did they actually support what this Todo was referring to?

I've lost almost all context hete

Reply:
As far as I can tell, yes. We are using it to add an additional action (loop on/off) that is not built-in to audio_service, which is what the issue was about if I'm not mistaken.

Reply:
Perfect, closed the issue then